content/about/notes/1.1/index.md

```diff
@@ -87,15 +92,6 @@ See [Istio Vault CA Integration](/docs/tasks/security/vault-ca) for more informa
 - TBD: How about adding [11667](https://github.com/istio/istio/issues/11667) as well? As this is also a significant feature for security which can enable end user to set
 different CA and Certs for different namespaces.
 
-## Multicluster
```

Comment:
Why remove this section?

@sdake @costinm

Reply:
coz I covered it already in my first rework of release notes, under traffic management